src/fmnoise/coldbrew.clj

```diff
@@ -102,6 +107,11 @@
   [^Cache cache key]
   (.invalidate cache key))
 
+(defn invalidate-keys
```

Comment:
I'm thinking if `invalidate-all` could be better name to mimic native call, but feels like `all` is a bit misleading because it's not "all" but listed ones 🤔

Reply:
What do you mean?

Reply:
I mean we're passing cache here, but what if we'll pass function, and this functions will extract cache using the implementation details